sdk/metricsadvisor/ai-metrics-advisor/review/ai-metrics-advisor.api.md

```diff
@@ -0,0 +1,1152 @@
+    refreshDataFeedIngestion(dataFeedId: string, startTime: Date, endTime: Date, options?: OperationOptions): Promise<RestResponse>;
+    updateAnomalyAlertConfiguration(alertConfigurationId: string, patch: AnomalyAlertConfigurationPatch, options?: OperationOptions): Promise<GetAnomalyAlertConfigurationResponse>;
+    updateDataFeed(dataFeedId: string, patch: DataFeedPatch, options?: OperationOptions): Promise<GetDataFeedResponse>;
+    updateHook(hookId: string, patch: EmailHookPatch | WebhookHookPatch, options?: OperationOptions): Promise<GetHookResponse>;
+    updateMetricAnomalyDetectionConfiguration(detectionConfigurationId: string, patch: AnomalyDetectionConfigurationPatch, options?: OperationOptions): Promise<GetAnomalyDetectionConfigurationResponse>;
+}
+
+// @public
+export interface MetricsAdvisorAdministrationClientOptions extends PipelineOptions {
+}
+
+// @public
+export class MetricsAdvisorClient {
+    constructor(endpointUrl: string, credential: MetricsAdvisorKeyCredential, options?: MetricsAdvisorClientOptions);
+    createMetricFeedback(feedback: MetricFeedbackUnion, options?: {}): Promise<GetFeedbackResponse>;
+    readonly endpointUrl: string;
+    getIncidentRootCauses(detectionConfigurationId: string, incidentId: string, options?: OperationOptions): Promise<GetIncidentRootCauseResponse>;
+    getMetricEnrichedSeriesData(detectionConfigurationId: string, startTime: Date, endTime: Date, seriesToFilter: DimensionKey[], options?: GetMetricEnrichedSeriesDataOptions): Promise<GetMetricEnrichedSeriesDataResponse>;
+    getMetricFeedback(feedbackId: string, options?: OperationOptions): Promise<GetFeedbackResponse>;
+    getMetricSeriesData(metricId: string, startTime: Date, endTime: Date, seriesToFilter: Record<string, string>[], options?: GetMetricSeriesDataOptions): Promise<GetMetricSeriesDataResponse>;
+    listAlertsForAlertConfiguration(alertConfigurationId: string, startTime: Date, endTime: Date, timeMode: TimeMode, options?: ListAlertsOptions): PagedAsyncIterableIterator<Alert, ListAlertsForAlertConfigurationPageResponse>;
+    listAnomaliesForAlert(alertConfigurationId: string, alertId: string, options?: ListAnomaliesForAlertConfigurationOptions): PagedAsyncIterableIterator<Anomaly, ListAnomaliesForAlertPageResponse>;
+    listAnomaliesForDetectionConfiguration(detectionConfigurationId: string, startTime: Date, endTime: Date, options?: ListAnomaliesForDetectionConfigurationOptions): PagedAsyncIterableIterator<Anomaly, ListAnomaliesForDetectionConfigurationPageResponse>;
+    listDimensionValuesForDetectionConfiguration(detectionConfigurationId: string, startTime: Date, endTime: Date, dimensionName: string, options?: ListDimensionValuesForDetectionConfigurationOptions): PagedAsyncIterableIterator<string, ListDimensionValuesForDetectionConfigurationPageResponse>;
```

Comment:
my eyes are starting to fuzz over from all these long identifier names, but I'm not sure how to shorten them

Reply:
Yeah I was following the pattern of MethodName + "Response". These are the page responses though so I feel it's better to also include "Page" in the name.  I hope it's not too much an issue since they are output type and customers don't use these types a lot?